python_src/src/lamp_py/performance_manager/flat_file.py

```diff
@@ -191,7 +288,7 @@ def generate_daily_table(
             ("headway_branch_seconds", pyarrow.int64()),
             ("service_date", pyarrow.int64()),
             ("route_id", pyarrow.string()),
-            ("direction_id", pyarrow.int8()),
+            ("direction_id", pyarrow.bool_()),
```

Comment:
in the before sql -> pandas -> pyarrow -> parquet flow this was being converted somewhere, but threw an exception when i ran in the sql -> pyarrow -> parquet flow.